ci/run.sh

```diff
@@ -54,6 +54,8 @@ if [ ! -z ${GG_BUILD_SYCL} ]; then
     fi
     export ONEAPI_DEVICE_SELECTOR="level_zero:0"
     export ZES_ENABLE_SYSMAN=1
+    # No plan to implement backward pass for now / disable test-opt
+    export GG_BUILD_LOW_PERF=1
```

Comment:
We should avoid to modify these build environment settings.

An alternative approach is to introduce `CTEST_EXTRA`, similar to `CMAKE_EXTRA` and use that to pass `-E test-opt` to `ctest` when SYCL is enabled.